openlibrary/plugins/openlibrary/js/editions-table/index.js

```diff
@@ -41,6 +41,37 @@ export function initEditionsTable() {
         localStorage.setItem(LS_RESULTS_LENGTH_KEY, length);
     });
 
+    $('#editions thead tr th').attr('tabindex', '0');
+    $('#editions thead tr th a').attr('tabindex', '-1');
```

Comment:
Hmm, would it work if we added the `tabindex="-1"` to the html here?

https://github.com/internetarchive/openlibrary/blob/dc927e5b007a81c555a58f1dacae0f07b0416433/openlibrary/templates/type/work/editions_datatable.html#L16-L17

If we put it there, I _think_ we won't need the keydown listener, since then the tabbing can behave like the browser default!